src/java.desktop/share/classes/javax/swing/plaf/basic/BasicTreeUI.java

```diff
@@ -3280,7 +3280,10 @@ public Rectangle getNodeDimensions(Object value, int row,
                      expanded, treeModel.isLeaf(value), row,
                      false);
                 if(tree != null) {
```

Comment:
```suggestion
                if (tree != null) {
```

Reply:
other places uses similar so kept it to keep it consistent..